spoon-client/src/main/java/com/squareup/spoon/Spoon.java

```diff
@@ -206,7 +205,7 @@ private static File filesDirectory(Context context, String directoryType, String
     File directory;
     if (Build.VERSION.SDK_INT >= 21) {
       // Use external storage.
-      directory = new File(getExternalStorageDirectory(), "app_" + directoryType);
+      directory = new File(context.getExternalFilesDir(null), "app_" + directoryType);
```

Comment:
We can switch to parsing the logs for files to pull. I looked at this a few years ago and it didn't seem hard, I was just lazy.